packages/graph/hash_graph/lib/graph/src/ontology/types/entity_type/links.rs

```diff
@@ -93,9 +92,9 @@ pub struct Links {
 }
 
 impl TryFrom<LinksRepr> for Links {
-    type Error = ValidationError;
+    type Error = crate::ontology::types::error::ValidationError;
```

Comment:
I guess this will also use `type_system::ValidationError` as soon as `LinkType`s were migrated?